src/components/books/book.css

```diff
@@ -90,6 +91,7 @@
 .list {
   display: flex;
   gap: 10px;
+  justify-content: flex-start;
 }
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/222924139-3f66847b-b8c2-4253-bba1-f2cf2d1120cb.png)

**[Optional]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalize to make your pull request more descriptive and professional 😊

Reply:
@HAMMAS-SALEEM , Thanks, I will improve.